usr/share/rear/layout/prepare/GNU/Linux/210_load_multipath.sh

```diff
@@ -38,7 +38,10 @@ blacklist {
     multipath >&2
     if [ $? -ne 0 ] ; then
         LogPrint "Failed to activate multipath, or no multipath device found."
-        rear_shell "Did you activate the multipath devices?"
+        user_input="$(UserInput -t 30 -p "Do you need to enter in rear_shell to manually activate multipath ? [type 'yes']")"
```

Comment:
I would write "Do you want to enter a shell in order to manually activate multipathing?" because `rear_shell` is an implementation detail irrelevant to the user.

An alternative would be to abort with a message prompting the user to fix the multipathing and run `rear recover` again. What is the value of giving the user shell access during an active recovery session?